src/server/index.js

```diff
@@ -46,12 +21,9 @@ function startServer(state) {
 
   // Generic server errors (e.g. not caught by components)
   server.use((err, req, res, next) => {
-    console.log("Error on request %s %s", req.method, req.url)
+    winston.error("Error on request " + req.method + " " + req.url + '\n'  + err.stack)
     if (server.get("env") === "production") {
-      console.log("err", err)
-      console.log("err.stack", err.stack)
-      sendErrorEmail(req, err)
-      res.status(500).send("Something unexpected happened, sorry. An email has been sent to the team.")
+      res.status(500).send("Something unexpected happened, sorry. The error has been logged.")
```

Comment:
I know this was already like that, but adding some information would be welcome: add `new Date().toISOString()` somewhere so that we can at least identify the information in the logs, and maybe even add `err`.